hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreScanner.java

```diff
@@ -635,8 +635,12 @@ public boolean next(List<Cell> outResult, ScannerContext scannerContext) throws
             scannerContext.incrementBatchProgress(1);
 
             if (matcher.isUserScan() && totalBytesRead > maxRowSize) {
-              throw new RowTooBigException(
-                  "Max row size allowed: " + maxRowSize + ", but the row is bigger than that.");
+              LOG.info("Max row size allowed: " + maxRowSize
+                + ", but the row is bigger than that, the row info: " + CellUtil
+                .toString(cell, true) + ", already have process row cells = " + outResult.size());
```

Comment:
Should we use LOG.warn or error here? Also, with variable params replacing `+ var +` with `{}` and argument?

Reply:
Is it necessary that we write out the log and throw the exception? With that we potentially have the same log entry twice.  Better would be to only have the exception with the full log message.

Reply:
@virajjasani  Done.

Reply:
For region name, we might need this log. Log is worth for `getRegionInfo().getRegionNameAsString()`. WDYT @HorizonNet ?

Reply:
@HorizonNet  regionserver log level default is INFO level and do not log the exception, the exception only return to client.

Reply:
@virajjasani  Yes, already log the regionname in the latest commit.

Reply:
The exception is not automatically logged?

Reply:
No, the exception is log only when RegionServer change log level to DEBUG/TRACE.

Reply:
Ok. Go ahead.